apps/s_server.c

```diff
@@ -873,7 +967,11 @@ const OPTIONS s_server_options[] = {
 
 #ifndef OPENSSL_NO_OCSP
     OPT_SECTION("OCSP"),
-    {"status", OPT_STATUS, '-', "Request certificate status from server"},
+    {"status", OPT_STATUS, '-',
+     "Provide certificate status response(s) (for the whole chain if possible) "
+     "if requested by client"},
+    {"status_server_only", OPT_OCSP_STATUS_SERVER, '-',
+     "Provide certificate status response only for server certificate"},
```

Comment:
The meaning of the options should be swapped, for backward compatibility.
So `-status` continues to deliver the status for the server cert only.
```suggestion
    {"status", OPT_STATUS, '-',
     "Provide certificate status response if requested, for server cert only"},
    {"status_all", OPT_OCSP_STATUS_ALL, '-',
     "Provide certificate status response(s) if requested, for the whole chain"},
```
Please adapt also the implementation and the documentation accordingly.